data/styles/clear/include/Basemap.mapcss

```diff
@@ -433,9 +433,15 @@ area|z10-[landuse=farmyard],
 
 /* Parking */
 
-area|z15-[amenity=parking]
+area|z15-[amenity=parking],
 {fill-color: @parking;}
+area|z15-[amenity=parking][parking=street_side],
+area|z15-[amenity=parking][parking=lane],
+{fill-color: ;}
```

Comment:
?
@pastk 
Better to set z15-[amenity=parking][!parking] ?

Reply:
We still want parking=surface etc to render at z15, would [!parking=street_side][!parking=lane] work?

Reply:
Nope, it won't work likes this...
Use `{fill-opacity: 0;}` and leave a comment its to remove the area drule. Ugly, but works :)
I'll improve it some day :)

Reply:
So you can do like 
```
area|z15[amenity=parking][parking=street_side],
area|z15[amenity=parking][parking=lane],
{fill-opacity: 0;}
```
to remove the fill from z15. A block below with the `fill-color: @parking` re-declaration is not needed.

Reply:
fixed